frontend/src/components/__tests__/WellTable.spec.js

```diff
@@ -40,7 +64,14 @@ describe('Rows of data give rows in the table', () => {
     expect(table.emitted().wellSelected[0][0]).toHaveProperty('idProduct')
     expect(table.emitted().wellSelected[0][0].idProduct).toEqual('ABCDEF')
 
-    await rows[2].find('button').trigger('click')
+    let wellButton = rows[2].find('button')
+    await wellButton.trigger('click')
     expect(table.emitted().wellSelected[1][0].idProduct).toEqual('123456')
+    expect(wellButton.classes('el-tooltip__trigger')).toBeTruthy()
```

Comment:
I think this is the best we can do with our current understanding. The tooltip doesn't emit any events for us to observe, at least documented ones. I see there is a "contentmenu" event, but there are also many others. It's hard to know what we would detect.